direct.go

```diff
@@ -25,6 +27,8 @@ const (
 	defaultMaxAllowedCachedAge = 24 * time.Hour
 	defaultMaxCacheSize        = 1000
 	defaultCacheSaveInterval   = 5 * time.Second
+	maxTries                   = 6
+	testURL                    = "http://d157vud77ygy87.cloudfront.net/measurements" // borda
```

Comment:
Good point, we don't really want to hammer borda. However, we need someplace to which we can actually POST. I could add a relatively dumb endpoint in borda that does nothing other than just accept a POST and return a 200 status. That way, the server at least wouldn't be doing much work.